src/App.js

```diff
@@ -32,11 +32,19 @@ function App() {
         <Route path='/signup' element={<Signup />} />
         <Route element={<PrivateRoute />}>
           <Route path='/home' element={<Home />} />
-          <Route path='/lable' element={<Lable />} />
+          <Route path='/label' element={<Label />} />
           <Route path='/archive' element={<Archive />} />
           <Route path='/trash' element={<Trash />} />
           <Route path='/profile' element={<Profile />} />
         </Route>
+        <Route
+          path='/home'
+          element={
```

Comment:
on line no. 34 , you are already doing the some, what's the need of repeating it

Reply:
It was by mistake